README.md

```diff
@@ -59,8 +61,8 @@ There is an example application provided in `example/` and you can run it with t
 1. `cd opentok-react/example/`
 1. `cp config.template.js config.js`
 1. Edit `config.js`:
-  1. Add your OpenTok API key, Session ID and Token (https://tokbox.com/account/)
-  1. Add your Chrome Extension ID (https://tokbox.com/developer/guides/screen-sharing/js/)
+1. Add your OpenTok API key, Session ID and Token (https://tokbox.com/account/)
```

Comment:
Hmm, this seems to be an issue with your editor? I think this is not letting markdown render as expected.